emv2/org.osate.aadl2.errormodel.faulttree.generation/src/org/osate/aadl2/errormodel/faulttree/generation/FTAGenerator.java

```diff
@@ -987,31 +994,31 @@ protected EObject processErrorEvent(ComponentInstance component, ErrorEvent erro
 
 	@Override
 	protected EObject postProcessAnd(ComponentInstance component, Element condition, ErrorTypes type,
-			double scale, List<EObject> subResults) {
+			List<EObject> subResults, double scale) {
 		Event ftaEvent = finalizeAsAndEvents(component, condition, type, subResults);
-		ftaEvent.setScale(scale);
+//		ftaEvent.setScale(scale);
 		return ftaEvent;
 	}
 
 	@Override
 	protected EObject postProcessXor(ComponentInstance component, Element condition, ErrorTypes type,
 			double scale, List<EObject> subResults) {
 		Event ftaEvent = finalizeAsXOrEvents(component, condition, type, subResults);
-		ftaEvent.setScale(scale);
+//		ftaEvent.setScale(scale);
 		return ftaEvent;
 	}
 
 	@Override
 	protected EObject postProcessOr(ComponentInstance component, Element condition, ErrorTypes type,
-			double scale, List<EObject> subResults) {
+			List<EObject> subResults, double scale) {
 		Event ftaEvent = finalizeAsOrEvents(component, condition, type, subResults);
-		ftaEvent.setScale(scale);
+//		ftaEvent.setScale(scale);
```

Comment:
Commented out code is suspicious. Did you mean to put that line back in before committing or should the line be deleted?